### ISSUE TITLE
Verbose for `razzle build`

### ISSUE THREAD

**Issue by heath-freenome** [jaredpalmer/razzle#1520]
## Razzle build with verbose webpack messages

I'm attempting refactor an existing application on top of razzle and the client build seems to take forever/hang. Is there a way to have the build display the webpack statuses using some sort of verbose flag so I can track where things are and how to stop a hang?

**Comment by fivethreeo** [comment]
In 4.0.0-canary.17

You can set options.debug.compile to true.

**Comment by heath-freenome** [comment]
@fivethreeo I tried this and I'm not seeing any verbose outputs during `npm run build`

My `razzle.config.js`:
```
module.exports = {
  options: {
    verbose: true,
    debug: {
      compile: true,
    },
  },
  modifyWebpackOptions({
  ... // more code
  });
... // more code
}
```

**Comment by fivethreeo** [comment]
In 4.0.0-canary.17?
https://github.com/jaredpalmer/razzle/blob/canary/packages/razzle/scripts/start.js#L33
https://github.com/jaredpalmer/razzle/blob/canary/packages/razzle/scripts/start.js#L152
https://github.com/jaredpalmer/razzle/blob/canary/packages/razzle/scripts/build.js#L207

**Comment by heath-freenome** [comment]
Yes...

I have specified the following in my `package.json`

```
  "devDependencies": {
    "babel-preset-razzle": "^4.0.0-canary.17",
...
    "razzle": "^4.0.0-canary.17",
    "razzle-dev-utils": "^4.0.0-canary.17",
...
}
```

**Comment by fivethreeo** [comment]
Will look into it tomorrow

**Comment by fivethreeo** [comment]
Fixed in v4.0.0-canary.18
